Root.py

```diff
@@ -2910,6 +2877,18 @@ def update_channel_bake_vcol(self, context):
     self['bake_vcol_force_first_index'] = True
     yp.bake_vcol_force_first_ch = yp.active_channel_index
 
+# Prevent vcol name from being null
+def get_channel_vcol_name(self):
+    if self['bake_vcol_name'] == '':
+        self['bake_vcol_name'] = 'Baked ' + self.name
+    return self['bake_vcol_name']
+
+def set_channel_vcol_name(self, value):
+    if value == '':
+        self['bake_vcol_name'] = 'Baked ' + self.name
+    else:
+        self['bake_vcol_name'] = value
```

Comment:
I've already provided the Getter and Setter for this variable, and it should work fine if the variable is empty😊.